packages/atlas/src/views/viewer/MemberView/MemberView.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { useParams } from 'react-router'
+import { useSearchParams } from 'react-router-dom'
+
+import { useMemberships } from '@/api/hooks'
+import { LimitedWidthContainer } from '@/components/LimitedWidthContainer'
+import { ViewErrorFallback } from '@/components/ViewErrorFallback'
+import { ViewWrapper } from '@/components/ViewWrapper'
+import { useUser } from '@/providers/user'
+import { SentryLogger } from '@/utils/logs'
+
+import { MemberAbout } from './MemberAbout'
+import { StyledMembershipInfo, StyledTabs, TabsContainer } from './MemberView.styles'
+
+const TABS = [
+  // 'NFTs',
+  // 'Activity',
+  'About',
+] as const
+
+export const MemberView: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams()
+  const currentTabName = searchParams.get('tab') as typeof TABS[number] | null
+  const [currentTab, setCurrentTab] = useState<typeof TABS[number] | null>(null)
+  const { activeMemberId } = useUser()
+  const { handle } = useParams()
+  const {
+    memberships,
+    error,
+    loading: loadingMember,
+  } = useMemberships(
+    { where: { handle_eq: handle } },
+    {
+      onError: (error) => SentryLogger.error('Failed to fetch memberships', 'ActiveUserProvider', error),
+    }
+  )
+  const member = memberships?.find((member) => member.handle === handle)
+
+  const handleSetCurrentTab = async (tab: number) => {
+    setSearchParams({ 'tab': TABS[tab] }, { replace: true })
+  }
+
+  const mappedTabs = TABS.map((tab) => ({ name: tab, badgeNumber: 0 }))
+  const tabContent = React.useMemo(() => {
+    switch (currentTab) {
+      case 'About':
+        return <MemberAbout />
+    }
+  }, [currentTab])
+
+  // At mount set the tab from the search params
+  const initialRender = useRef(true)
+  useEffect(() => {
+    if (initialRender.current) {
+      const tabIndex = TABS.findIndex((t) => t === currentTabName)
+      if (tabIndex === -1) setSearchParams({ 'tab': TABS[0] }, { replace: true })
+      initialRender.current = false
+    }
+  })
+
+  useEffect(() => {
+    if (currentTabName) {
+      setCurrentTab(currentTabName)
+    }
+  }, [currentTabName])
+
+  if (error) {
+    return <ViewErrorFallback />
+  }
```

Comment:
I think we don't handle the member not found in any way?

Reply:
Added the usual we do for that